task_processing/plugins/mesos/execution_framework.py

```diff
@@ -140,11 +135,13 @@ def _background_check(self):
                                 task_event(
                                     task_id=task_id,
                                     terminal=True,
-                                    timestamp=time.time(),
+                                    timestamp=time_now,
                                     success=False,
                                     message='stop',
                                     task_config=md.task_config,
-                                ))
+                                    raw='Failed due to offer timeout',
```

Comment:
Can you change this to addict.Dict to make it consistent with other places?

Reply:
as this field is dedicated for platform-specific event data it can end up being any datatype, depending on the backend or even mesos library version. we shouldn't give anyone ideas that they can rely on form of this value.